app/src/main/java/com/arygm/quickfix/ui/uiMode/appContentUI/userModeUI/camera/QuickFixDisplayImages.kt

```diff
@@ -50,18 +51,23 @@ import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
 import com.arygm.quickfix.R
+import com.arygm.quickfix.model.offline.small.PreferencesViewModel
 import com.arygm.quickfix.model.search.AnnouncementViewModel
+import com.arygm.quickfix.model.switchModes.AppMode
 import com.arygm.quickfix.ui.navigation.NavigationActions
 import com.arygm.quickfix.ui.theme.poppinsTypography
+import com.arygm.quickfix.utils.loadAppMode
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun QuickFixDisplayImages(
-    canDelete: Boolean = true,
     navigationActions: NavigationActions,
+    preferencesViewModel: PreferencesViewModel,
     announcementViewModel: AnnouncementViewModel,
     images: List<Bitmap> = emptyList() // For testing
 ) {
+  var canDelete by remember { mutableStateOf(true) }
+  LaunchedEffect(Unit) { canDelete = loadAppMode(preferencesViewModel) == AppMode.USER.name }
```

Comment:
you could create a flow for this preference as i did with is worker and all the other fields of the account cached, but is ok also.